lib/template/spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 #
-ENV["RACK_ENV"] = "test"
+ENV["PLINY_ENV"] = "test"
```

Comment:
I discussed quickly with @rkh first, who agreed on the change, but for 2.0 only.